typescript/types.ts

```diff
@@ -72,8 +72,31 @@ export type SchemaVersion =
   | "v1"
   | "latest";
 
+export type Attachment = {
+  /**
+   * The data representing the attachment
+   */
+  data: any;
+
+  /**
+   * The MIME type of the result. If not specified, the MIME type will be
+   * assumed to be plain text
+   */
+  mime_type: string | undefined;
+
+  /**
+   * Attachment metadata
+   */
+  metadata?: JSONValue;
```

Comment:
Should be object to match elsewhere

```suggestion
  metadata?: {
    [k: string]: any;
  };
```